src/SFA.DAS.Rofjaa.Application/Models/PagedQuery.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.Rofjaa.Application.Models
+{
+    public abstract class PagedQuery
```

Comment:
Are we using this? I didn't notice any paging in the handlers - I suggest we don't, and probably never will due to the small scale of the data.

Reply:
Good thinking, left in for the future but agreed unlikely would ever need, refactored and renamed to QueryResult<T> to contain a List of Items as a collection is needed to return but nuked all references to anything Paging related.